test/helpers/env.ts

```diff
@@ -0,0 +1,36 @@
+import { env } from 'process';
+
+type HeadlessCapableServiceName = 'SELENIUM' | 'PLAYWRIGHT';
+
+export function isHeadless(serviceName: HeadlessCapableServiceName): boolean {
+  if (serviceName) {
+    const serviceKey = `${serviceName}_HEADLESS`;
+    if (env[serviceKey]) {
+      return parseBoolean(env[serviceKey]);
+    }
+  }
+  return !!env.HEADLESS && parseBoolean(env.HEADLESS);
+}
+
+export function parseBoolean(value: undefined | string): boolean {
+  if (!value) {
+    return false;
+  }
+  if (typeof value === 'boolean') {
+    return value;
+  }
+  if (typeof value !== 'string') {
+    throw new Error(`Not-a-Boolean: '${value}'`);
+  }
+  switch(value.toLowerCase().trim()) {
+    case 'false':
+    case '0':
+    case '':
```

Comment:
This could actually be removed as it's handled by the initial `!value` check - but I also figure it doesn't hurt to be a bit overly explicit. Happy to remove if the redundancy disturbs anyone! ;)